flink-connectors/flink-connector-cassandra/src/main/java/org/apache/flink/streaming/connectors/cassandra/CassandraSink.java

```diff
@@ -96,18 +98,19 @@ private CassandraSink(SingleOutputStreamOperator<IN> sink) {
 
 	/**
 	 * Sets an additional, user provided hash for this operator.
-	 *
 	 * <p/>
 	 * <p>The user provided hash is an alternative to the generated hashes, that is considered when identifying an
 	 * operator through the default hash mechanics fails (e.g. because of changes between Flink versions.
```

Comment:
Ok, I misunderstood what closing bracket we were talking about :-). Fixed those typos.